### ISSUE TITLE
Can't resolve '@appsignal/core' in ... node_modules/@appsignal/javascript/dist/esm

### ISSUE THREAD

**Issue by hubertlepicki** [appsignal/appsignal-javascript#181]
I am updating the JavaScript AppSignal client library to version 1.1.0, from version 1.0.1.

It's a pretty standard phoenix/webpack project. The library gets updated correctly but when attempting to build project I get the following error:

```
...
    + 157 hidden modules

ERROR in ./node_modules/@appsignal/javascript/dist/esm/index.js
Module not found: Error: Can't resolve '@appsignal/core' in '/home/hubert/Projekty/myproject/myproject-phoenix/apps/ui/assets/node_modules/@appsignal/javascript/dist/esm'
 @ ./node_modules/@appsignal/javascript/dist/esm/index.js 2:0-53 55:12-19 64:12-19 135:119-128
 @ ./js/app.js
 @ multi ./js/app.js

ERROR in ./node_modules/@appsignal/javascript/dist/esm/api.js
Module not found: Error: Can't resolve '@appsignal/core' in '/home/hubert/Projekty/myproject/myproject-phoenix/apps/ui/assets/node_modules/@appsignal/javascript/dist/esm'
 @ ./node_modules/@appsignal/javascript/dist/esm/api.js 3:0-44 39:33-42
 @ ./node_modules/@appsignal/javascript/dist/esm/index.js
 @ ./js/app.js
 @ multi ./js/app.js

ERROR in ./node_modules/@appsignal/javascript/dist/esm/span.js
Module not found: Error: Can't resolve '@appsignal/core' in '/home/hubert/Projekty/myproject/myproject-phoenix/apps/ui/assets/node_modules/@appsignal/javascript/dist/esm'
 @ ./node_modules/@appsignal/javascript/dist/esm/span.js 2:0-79 32:23-36 37:66-81 49:2-14
 @ ./node_modules/@appsignal/javascript/dist/esm/index.js
 @ ./js/app.js
 @ multi ./js/app.js
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js!node_modules/postcss-loader/src/index.js!css/app.css:
    Entrypoint mini-css-extract-plugin = *
    [1] ./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/src!./css/app.css 1.02 MiB {0} [built]
    [3] ./node_modules/css-loader/dist/cjs.js!./css/flatpickr-light.css 3.76 KiB {0} [built]
    [6] ./node_modules/css-loader/dist/cjs.js!./css/headway.css 441 bytes {0} [built]
        + 4 hidden modules
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @ deploy: `webpack --mode production`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @ deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/hubert/.npm/_logs/2020-01-29T16_23_13_098Z-debug.log
```

I have this in my package.json:

```
 ...
  "dependencies": {
    "@appsignal/javascript": "^1.1.0",
    "@appsignal/plugin-window-events": "^1.0.0",
    "@appsignal/stimulus": "^1.0.0",
  ...
```

Something off with that version update or maybe conflicts with plugins?

**Comment by xadamy** [comment]
Thanks for reporting @hubertlepicki, I've just pushed v1.1.1 to fix this issue (08cd81e) so please bump your `package.json` accordingly! 🎉

**Comment by hubertlepicki** [comment]
Thanks for a quick solution! Works now.
